### ISSUE TITLE
Missing dependency

### ISSUE THREAD

**Issue by arranstewart** [jjfiv/quizdown#7]
The Python package for quizdown seems not to be declaring its dependencies correctly.

**Version information:**

Using Python 3.6.9 on Ubuntu 18.04 and current version of quizdown (0.3.0).

**Steps to reproduce**

```
$ python3 -m venv env
$ . env/bin/activate
$ pip install quizdown
$ python -m quizdown --help
```

**Expected behaviour**

Help message should be displayed.

**Actual behaviour**

```
(env) $ python -m quizdown --help
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.6/runpy.py", line 142, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/lib/python3.6/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/home/user/env/lib/python3.6/site-packages/quizdown/__init__.py", line 2, in <module>
    from .types import Quiz, Question, QOption
  File "/home/user/env/lib/python3.6/site-packages/quizdown/types.py", line 5, in <module>
    @attr.s
AttributeError: module 'attr' has no attribute 's'
```

**Comments**

This error can be fixed by installing the `attrs` package as well as quizdown:

```
$ pip install attrs quizdown
```

But quizdown should correctly state its dependencies, and appears not to. (Double-checked by looking at the `quizdown` page on PyPi, <https://pypi.org/project/quizdown/> – there's no "Requires" section at all.

**Comment by jjfiv** [comment]
Now that I’m getting back into it, I think I just need to push a new version. Sadly my GitHub actions have bit rotted, eg #8.

**Comment by jjfiv** [comment]
This should be fixed in 0.3.2. Feel free to reopen or report otherwise.
